SprawdzPogode/UserConfig.xml

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<config>
+  <files>
+    <file type="input">\\spplfapcen02\userdata\CE50NV\Documents\Exercise\SprPogode\Lista.txt</file>

```

Comment:
This should not be present inside Git, please:
1. Add `SprawdzPogode/UserConfig.xml` to the end of `.gitignore`
2. Create new file called `UserConfig.xml.dist` which will have some random paths inside "input" and "output".

Update Readme.md with instruction that the person whom wishes to use program must copy over UserConfig.xml.dist as UserConfig.xml and fill it with their data.